lab_3_keywords_textrank/main.py

```diff
@@ -101,7 +89,9 @@ def preprocess_text(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens with no stop-words
         """
-        return self._remove_stop_words(self._clean_and_tokenize(text))
+        tokens = self._clean_and_tokenize(text)
+        filtered_tokens = self._remove_stop_words(tokens)
+        return filtered_tokens
```

Comment:
you can return the result of `_remove_stop_words` without saving to a variable

Reply:
done